ios/RCTFBSDK/login/RCTFBSDKLoginManager.m

```diff
@@ -90,14 +90,27 @@ + (BOOL)requiresMainQueueSetup
                                                     tracking: tracking
          ];
     }
-    
-    
+
+
```

Comment:
not sure about the unrelated spacing changes, in most repositories I run I use the `clang-format` npm package to just enforce a consistent styles on the Obj-C code so no one has to think about it really (other than to turn the formatter) but not sure if I integrated it here, CI will tell the tale

Reply:
No problem with this one it appears I don't have clang-format integrated, and that's not on my priority list at the moment. Noted a typescript lint issue below though